daml-assistant/get-daml.sh

```diff
@@ -26,13 +26,23 @@ else
 fi
 cd $TMPDIR
 
+# Don't remove user specified temporary directory on cleanup.
+rmTmpDir() {
+  if [ -z $TEMPDIR ]; then
+    rm -rf $TMPDIR
+  else
+    echo "Feel free to remove DAML files in $TEMPDIR."
```

Comment:
Ok, this is perhaps too blasé 😅 How about . . .

```suggestion
    echo "You may now remove the DAML installation files from $TEMPDIR"
```